plugins/tabletools/dialogs/tableCell.js

```diff
@@ -48,18 +49,110 @@ CKEDITOR.dialog.add( 'cellProperties', function( editor ) {
 		};
 	}
 
-	// Reads the unit of width property of the table cell.
+	// Reads the unit of a CSS length.
 	//
-	// * @param {CKEDITOR.dom.element} cell An element representing table cell.
-	// * @returns {String} A unit of width: 'px', '%' or undefined if none.
-	function getCellWidthType( cell ) {
-		var match = widthPattern.exec(
-			cell.getStyle( 'width' ) || cell.getAttribute( 'width' ) );
-
+	// * @param {String} length the length with a unit.
+	// * @returns {String} A unit of length: 'px', 'cm', 'mm', 'pt', '%' or undefined if none.
+	function getLengthUnit( length ) {
+		var match = lengthUnitPattern.exec( length );
 		if ( match )
 			return match[ 2 ];
 	}
 
+	// Reads the unit of a length property of the table cell.
+	//
+	// * @param {String} cssProperty The name of a length CSS property (width, height, padding-left, etc.)
```

Comment:
Kudos for paying attention to code docs at a time of writing, this makes things so much easier! 👍